Makefile

```diff
@@ -30,8 +30,6 @@
 .PHONY: all
 all: test
 
-include $(shell cocotb-config --makefiles)/Makefile.inc
-
```

Comment:
Let's put this line back and merge it as it was.

Reply:
I will look at this at some point.

Reply:
It seems that `LD_LIBRARY_PATH` is not correct if we don't have the include here. It works on my machine because I have cocotb installed in editable mode, so there is a build dir.

Fails, pointing locally: https://travis-ci.org/github/cocotb/cocotb/jobs/664893296#L628
Passes, points into install dir: https://travis-ci.org/github/cocotb/cocotb/jobs/664959814#L628

Reply:
I will try to look at this. I hope LD_LIBRARY_PATH will go away soon.